lib/screenplay/pa_messages/pa_message/queries.ex

```diff
@@ -21,11 +21,35 @@ defmodule Screenplay.PaMessages.PaMessage.Queries do
 
   A PaMessage is considered "active" if its start time is in the past and
   either its end time is in the future or it has no end time and its associated
+  alert is in passed list of alert IDs and is also not paused.
+  """
+  @spec active(
+          queryable :: Ecto.Queryable.t(),
+          alert_ids :: [String.t()],
+          now :: DateTime.t()
+        ) ::
+          Ecto.Query.t()
+  def active(q \\ PaMessage, alert_ids, now) do
+    current_service_day_of_week = Util.get_current_service_day(now)
+
+    from m in q,
+      where:
+        ^current_service_day_of_week in m.days_of_week and
+          m.start_datetime <= ^now and
+          ((is_nil(m.end_datetime) and m.alert_id in ^alert_ids) or m.end_datetime >= ^now) and
```

Comment:
A big chunk of this clause looks the same as the one in `current`. Can we use composition somehow to reuse that?